src/Microsoft.SqlTools.Migration/MigrationService.cs

```diff
@@ -611,15 +611,33 @@ internal async Task HandleGetArmTemplateRequest(
         {
             try
             {
-                Logger.Verbose("request received in toolsservice");
```

Comment:
I would suggest to keep log statements as they help in debugging customer issues

Reply:
Will create a separate task fo logging. None of our APIs are adding logs as of now. May be we can have one PR which adds logs for all of APIs